cdap-client-tests/src/test/java/co/cask/cdap/client/ApplicationClientTestRun.java

```diff
@@ -169,14 +169,15 @@ public void testAppUpdate() throws Exception {
     Id.Artifact artifactIdV1 = Id.Artifact.from(Id.Namespace.DEFAULT, artifactName, "1.0.0");
     Id.Artifact artifactIdV2 = Id.Artifact.from(Id.Namespace.DEFAULT, artifactName, "2.0.0");
     Id.Application appId = Id.Application.from(Id.Namespace.DEFAULT, "ProgramsApp");
-    try {
-      artifactClient.add(Id.Namespace.DEFAULT, artifactName,
-        Files.newInputStreamSupplier(createAppJarFile(ConfigurableProgramsApp.class)),
-        "1.0.0");
-      artifactClient.add(Id.Namespace.DEFAULT, artifactName,
-        Files.newInputStreamSupplier(createAppJarFile(ConfigurableProgramsApp2.class)),
-        "2.0.0");
 
+    artifactClient.add(Id.Namespace.DEFAULT, artifactName,
```

Comment:
not changed in this PR, but would you agree that artifactClient should take in `Id.Artifact` instead of namespace, artifactName, and version?